tests/test_config_parsing/test_ensemble_config.py

```diff
@@ -43,24 +45,25 @@ def test_ensemble_config_errors_on_unknown_function_in_field(config_dict):
 @pytest.mark.skip(reason="github.com/equinor/ert/issues/4071")
 @pytest.mark.usefixtures("use_tmpdir")
 @given(config_dicts())
-def test_ensemble_config_errors_on_double_field(config_dict):
-    # TODO in the logs i see a doubled OUT_FILE, not two fields with equal
-    # name, as i first suspected - figure out what's going on here!
+def test_ensemble_config_errors_on_identical_name_for_two_enkf_config_nodes(
+    config_dict,
+):
     filename = config_dict[pytest.TEST_CONFIG_FILE_KEY]
-    if ConfigKeys.FIELD_KEY not in config_dict:
-        return
-    # TODO manipulate FIELD to be degenerate in the desired way
+    two_keys = [ConfigKeys.FIELD_KEY, ConfigKeys.GEN_DATA]
+    for key in two_keys:
+        assume(key in config_dict and len(config_dict[key]) > 0)
+    one_node = config_dict[two_keys[0]][0]
+    other_node = config_dict[two_keys[1]][0]
+    one_node[ConfigKeys.NAME] = other_node[ConfigKeys.NAME]
+
     to_config_file(filename, config_dict)
     with pytest.raises(
-        expected_exception=ValueError, match="TODO this should be a meaningful message"
+        expected_exception=ConfigValidationError,
+        match="TODO this should be a meaningful message",
```

Comment:
Could you remove the "TODO" match. I think we can leave it to the implementer to just fill it out, or we could match for "duplicate key"

Reply:
for sure. it is done.